g4f/Provider/base_provider.py

```diff
@@ -108,12 +107,17 @@ def create_async_generator(
     ) -> AsyncGenerator:
         raise NotImplementedError()
 
-# Don't create a new loop in a running loop
-def check_running_loop():
+
+def create_event_loop():
+    # Don't create a new loop in a running loop
     if asyncio.events._get_running_loop() is not None:
         raise RuntimeError(
             'Use "create_async" instead of "create" function in a async loop.')
 
+    # Force use selector event loop on windows
+    return asyncio.SelectorEventLoop()
```

Comment:
Have you tried something like 
`if sys.platform.startswith('win32'):
    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())` 
and use asyncio.run() in create() instead? Seems like it would abstract the platform better that way

Reply:
Good point, didn't think of that